Compilation fails: missing `bitset` include

Building (with GCC 11.4) fails:
```cpp
FCIQMC/runFCIQMC.cpp: In function ‘void calcEN2Correction(walkersFCIQMC<TrialWalk>&, const spawnFCIQMC&, const oneInt&, const twoInt&, double&, double, double, double, double&)’:
FCIQMC/runFCIQMC.cpp:560:7: error: ‘bitset’ was not declared in this scope
  560 |       bitset<max_nreplicas> initFlags(spawn.flags[j]);
      |       ^~~~~~
FCIQMC/runFCIQMC.cpp:16:1: note: ‘std::bitset’ is defined in header ‘<bitset>’; did you forget to ‘#include <bitset>’?
   15 | #include "trivialWF.h"
  +++ |+#include <bitset>
   16 | #include "trivialWalk.h"
FCIQMC/runFCIQMC.cpp:561:14: error: ‘initFlags’ was not declared in this scope
  561 |       if ( (!initFlags.test(0)) && (!initFlags.test(1)) ) {
      |              ^~~~~~~~~
FCIQMC/runFCIQMC.cpp: In instantiation of ‘void calcEN2Correction(walkersFCIQMC<TrialWalk>&, const spawnFCIQMC&, const oneInt&, const twoInt&, double&, double, double, double, double&) [with TrialWalk = TrivialWalk]’:
FCIQMC/runFCIQMC.cpp:398:26:   required from ‘void runFCIQMC(Wave&, TrialWalk&, int, int, int, int) [with Wave = TrivialWF; TrialWalk = TrivialWalk]’
FCIQMC/runFCIQMC.cpp:867:58:   required from here
FCIQMC/runFCIQMC.cpp:560:38: error: ‘initFlags’ was not declared in this scope
  560 |       bitset<max_nreplicas> initFlags(spawn.flags[j]);
      |                             ~~~~~~~~~^~~~~~~~~~~~~~~~
FCIQMC/runFCIQMC.cpp: In instantiation of ‘void calcEN2Correction(walkersFCIQMC<TrialWalk>&, const spawnFCIQMC&, const oneInt&, const twoInt&, double&, double, double, double, double&) [with TrialWalk = Walker<Jastrow, AGP>]’:
FCIQMC/runFCIQMC.cpp:398:26:   required from ‘void runFCIQMC(Wave&, TrialWalk&, int, int, int, int) [with Wave = CorrelatedWavefunction<Jastrow, AGP>; TrialWalk = Walker<Jastrow, AGP>]’
FCIQMC/runFCIQMC.cpp:872:58:   required from here
FCIQMC/runFCIQMC.cpp:560:38: error: ‘initFlags’ was not declared in this scope
FCIQMC/runFCIQMC.cpp: In instantiation of ‘void calcEN2Correction(walkersFCIQMC<TrialWalk>&, const spawnFCIQMC&, const oneInt&, const twoInt&, double&, double, double, double, double&) [with TrialWalk = Walker<Jastrow, Slater>]’:
FCIQMC/runFCIQMC.cpp:398:26:   required from ‘void runFCIQMC(Wave&, TrialWalk&, int, int, int, int) [with Wave = CorrelatedWavefunction<Jastrow, Slater>; TrialWalk = Walker<Jastrow, Slater>]’
FCIQMC/runFCIQMC.cpp:877:58:   required from here
FCIQMC/runFCIQMC.cpp:560:38: error: ‘initFlags’ was not declared in this scope
FCIQMC/runFCIQMC.cpp: In instantiation of ‘void calcEN2Correction(walkersFCIQMC<TrialWalk>&, const spawnFCIQMC&, const oneInt&, const twoInt&, double&, double, double, double, double&) [with TrialWalk = SimpleWalker]’:
FCIQMC/runFCIQMC.cpp:398:26:   required from ‘void runFCIQMC(Wave&, TrialWalk&, int, int, int, int) [with Wave = SelectedCI; TrialWalk = SimpleWalker]’
FCIQMC/runFCIQMC.cpp:881:58:   required from here
FCIQMC/runFCIQMC.cpp:560:38: error: ‘initFlags’ was not declared in this scope
```
It seems that `FCIQMC/runFCIQMC.cpp` and `FCIQMC/spawnFCIQMC.cpp` miss this include:
```c++
#include <bitset>
```

--- a/FCIQMC/runFCIQMC.cpp
+++ b/FCIQMC/runFCIQMC.cpp
@@ -1,4 +1,5 @@
 #include <iomanip>
+#include <bitset>
 #include "Determinants.h"
 #include "evaluateE.h"
 #include "global.h"

--- a/FCIQMC/spawnFCIQMC.cpp
+++ b/FCIQMC/spawnFCIQMC.cpp
@@ -19,6 +19,7 @@
 
 #include <iostream>
 #include <vector>
+#include <bitset>
 #include <boost/serialization/serialization.hpp>
 #include <Eigen/Dense>
 #include "global.h"

